code/src/main/webapp/CSS/style.css

```diff
@@ -162,16 +162,16 @@ div {
   transition-delay: 1s
 }
 
-#mission-statement {
+.mission-statement {
```

Comment:
any reason these are classes and not ids anymore? (for all of the ones you changed)

Reply:
id should be unique identifier so I changed repeated ids to classes.